modules/insights/action-groups/README.md

```diff
@@ -1,4 +1,4 @@
-# Action Groups `[Microsoft.Insights/actionGroups]`
```

Comment:
That's a bit off an odd one. In the specs file, it is upper case, yet here it is lower. Not sure, if it is worth troubleshooting, but it happened in a few places.

Reply:
I just noticed it is now fixed in main, which is great. It's lowecase in my branch specs file and that's the reason of the changes. I'll merge latest and regen the Microsoft.Insight readmes later.

Reply:
How strange, given that the file is auto-updated in main. Sounds almost like the API changed for like a few days :D

Reply:
![image](https://github.com/Azure/ResourceModules/assets/56914614/55002d1b-1b17-419d-8822-e9ce96392680)

Looks like [it](https://github.com/Azure/ResourceModules/commit/557998bd46373e82282817a199276f532fbaf6b7#diff-b17a07fc484e846ee3d3e558767b0b068c628eb15bd8c3d4338fb79d02011623L17943)
Anyway updated 🔨